pkg/languages/js/js.go

```diff
@@ -130,19 +133,20 @@ func (j jsLanguageManager) DownloadRuntimeIfRequired() error {
 	return nil
 }
 
-func (r jsLanguageManager) ConfigureEditingDirectory(mainFile string, functionConfiguration map[string][]string, editingDirectory string) (string, string, error) {
+// ConfigureEditingDirectory does something
+func (j jsLanguageManager) ConfigureEditingDirectory(mainFile string, functionConfiguration map[string][]string, editingDirectory string) (string, string, error) {
 	functionFile := path.Join(editingDirectory, "index.js")
 	err := util.Link(mainFile, functionFile)
 	if err != nil {
 		return "", "", err
 	}
 
-	packageJson, err := generatePackageJson(functionConfiguration)
+	packageJSON, err := getPackageJSON(editingDirectory, functionConfiguration)
```

Comment:
If the package json exists, it should be linked and not copied in editing dir. Otherwise the user modifies the package json but these changes are lost